grpc-sys/src/grpc_wrap.rs

```diff
@@ -0,0 +1,94 @@
+use super::*;
+
+use std::slice;
+use std::time::Duration;
+
+pub const GRPC_INITIAL_METADATA_IDEMPOTENT_REQUEST: u32 = 0x0000_0010;
```

Comment:
Aren't these consts part of generated bindings?

Reply:
Solved